src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
I do not really understand logic of extending `ColorSupplier`. Is lottery always with colors? I think it is better to use a composition instead of inheritance (when ColorSupplier is dynamic field of Lottery class. So that we can add much more different suppliers there in future).